website/docs/docs/use-dbt-semantic-layer/sl-architecture.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "dbt Semantic Layer product architecture"
+id: sl-architecture
+description: "dbt Semantic Layer product architecture and related questions."
+sidebar_label: "Product architecture"
+tags: [Semantic Layer]
+---
+
+<VersionBlock firstVersion="1.6">
+
+At a high level, the dbt Semantic Layer allows you to define metrics and query them using various interfaces. The Semantic Layer does the heavy lifting to find where the queried data exists in your data platform and generates the SQL to make the request (including performing joins). This is possible with the following components:
```

Comment:
Can we remove "at a high level"? I also got caught up with what we were using various interfaces for (I originally thought it was to query and to define metrics until I read the API description!) Let me know how this rewrite reads!
```suggestion
The dbt Semantic Layer allows you to define metrics and use various interfaces to query them. The Semantic Layer does the heavy lifting to find where the queried data exists in your data platform and generates the SQL to make the request (including performing joins). The Semantic Layer includes these components:
```